extensions/amp-image-slider/0.1/amp-image-slider.css

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
- .i-amphtml-image-slider-container {
+.i-amphtml-image-slider-container {
```

Comment:
We may want to `!important` everything that we do not want to make customizable, since there are ways that developers can manipulate them with node / child / sibling selectors.